Python/assemble.c

```diff
@@ -467,19 +467,19 @@ compute_localsplus_info(_PyCompile_CodeUnitMetadata *umd, int nlocalsplus,
     int nlocals = (int)PyDict_GET_SIZE(umd->u_varnames);
 
     // This counter mirrors the fix done in fix_cell_offsets().
-    int numdropped = 0;
+    int numdropped = 0, cellvar_offset = -1;
```

Comment:
```suggestion
    int numdropped = 0;
```

Reply:
Or actually, I think you could not declare a new variable on line 478 before, so `cellvar_offset` is still available for the assertion further down.

However, I think it may be even better to undo all changes in this file. There's no changed functionality, and it's better to make changes as small as possible in the bugfix branches.